.github/workflows/build.yml

```diff
@@ -0,0 +1,77 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Does this mean that we do not want to test pull requests until they are merged to master?

Reply:
No it means that it'll only test pull requests targeting master.